setup.py

```diff
@@ -41,7 +41,7 @@
     # TODO Unpin JAX, once new rlax and acme version is released.
     # To fix cannot import name 'tree_multimap' from 'jax.tree_util'.
     "jax<=0.3.15",
-    "jaxlib",
+    "jaxlib<=0.3.19",
```

Comment:
Thanks @OmaymaMahjoub !  We should pin jax and jaxlib to the same versions.

Reply:
Okay! Should we keep jax to 0.3.15 or change it to 0.3.17 since it works fine in that version too?

Reply:
I updated it :) . Please let me know if you are happy?

Reply:
Thank you! I think the latest jax is 0.3.18 and it didn't work in our case bringing some errors so probably we can just stay with 0.3.17 so let's wait to see if the checks are fine else we can re edit it

Reply:
It's working fine 🙂 thank you 🙌